### DISCUSSION TITLE
js/js_browser build target outputs ~6k LOC for a hello world

### DISCUSSION THREAD

**Discussion by danawoodman** [vlang/v#19296] [General]
For the following V code:

```v
module main

fn main() {
	println('Hello, world!')
}
```

...building with `v -b js_browser .` I get nearly 6k LOC:

![image](https://github.com/vlang/v/assets/157695/f6c8c62a-6f01-4359-9ac4-d4037a3c8cea)

...using `v -b js_browser -skip-unused -prod .` it improves a lot, but ends up being ~1800 LOC:

![image](https://github.com/vlang/v/assets/157695/aa07be59-e2ff-4ef4-b70f-18eb7838efa4)

I'm assuming this is planned to be improved over time, but is there something I'm missing to enable it to output a more reasonable file?

**Comment by JalonSolov** [comment]
Yes, it will be improved.  No, you're not missing anything right now.
